account_banking_partner_journal_account/__openerp__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Therp BV (<http://therp.nl>) / BAS Solutions
+#    All Rights Reserved
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Default partner journal accounts for bank transactions',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'author': "Therp BV,BAS Solutions,Odoo Community Association (OCA)",
+    'category': 'Banking',
+    'depends': ['base', 'account'],
+    'description': '''
+This module allows to set alternative journal accounts on partners to use
```

Comment:
Please extract this description to a README.rst file with OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst